README.md

```diff
@@ -4,31 +4,35 @@
 
 Welcome to this repo!
 
-In this repo you will learn about Haraj Solutions Team's secret technique in writing maintainable code for Haraj projects. 🥷🏻💥
+In this repo you will learn about Haraj Solutions Team's secret technique in writing production-grade code for Haraj projects that help serves millions of users daily in Saudi Arabia. 🥷🏻💥
 
 The secret technique actually lies in the software architecture we choose for these projects: [Hexagonal Architecture](./docs/reference/hex-architecture.md).
 
-In this repo we will share to you our knowledge on this special architecture through simple server-client game named `Hex Monscape`. We are using [Hexagonal Architecture](./docs/reference/hex-architecture.md) to structure the game server code while implementing it using [Go](https://go.dev/) & for the web client we implement it using [Vue 3](https://vuejs.org/).
+Since this repo is intended as getting started guidelines for anyone who wants to know how we write production code for Haraj projects, we will try our best to explain the technique in simple & fun ways. 
+
+In order to do that, we will be using simple server-client game named `Hex Monscape`. We are using [Hexagonal Architecture](./docs/reference/hex-architecture.md) to structure the game server code while implementing it using [Go](https://go.dev/). For the web client we implement it using [Vue 3](https://vuejs.org/).
```

Comment:
The added description is giving more clarity to people what we intended to share with this repo. But I think, we also need more emphasis onto how people will be more interest to read this complete project as a whole.

Game is one point that catches many people interest. I think we should label this repo more of a "Simple Game for Haraj Engineering Onboarding", so people will have something in mind like "Oooh, they learn to write a game when onboarding? Interesting, I need to check this out".

Giving more catchy introduction like a screenshot of the game itself even better I think, to make people stay and read the repo. These are my two cents @riandyrn  😄